project/LinkerdBuild.scala

```diff
@@ -257,6 +257,16 @@ object LinkerdBuild extends Base {
        |   -Dio.netty.allocator.numHeapArenas=${FINAGLE_WORKERS:-8}      \
        |   -Dio.netty.allocator.numDirectArenas=${FINAGLE_WORKERS:-8}    \
        |   -Dcom.twitter.finagle.netty4.numWorkers=${FINAGLE_WORKERS:-8} \
+       |   -XX:+PrintGCDetails                                           \
+       |   -XX:+PrintGCDateStamps                                        \
+       |   -XX:+PrintHeapAtGC                                            \
+       |   -XX:+PrintTenuringDistribution                                \
+       |   -XX:+PrintGCApplicationStoppedTime                            \
+       |   -XX:+PrintPromotionFailure                                    \
+       |   -Xloggc:/var/log/linkerd/gc.log                               \
```

Comment:
do you think we should use an env var here so this is overridable?

Reply:
Good point. I'll get that set up in the next commit.